src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -1130,6 +1138,23 @@ public void CheckAndMarkAsPageable(DottedNameNode node)
             }
         }
 
+        public void CheckAndMarkAsDelegatable(FirstNameNode node)
+        {
+            Contracts.AssertValue(node);
+            Contracts.AssertIndex(node.Id, _typeMap.Length);
+
+            if (SupportsDelegation(node))
+            {
+                _isDelegatable.Set(node.Id, true);
```

Comment:
For consistency, we should stick to one of "delegatable" or "delegable",

Reply:
since this is `isDelegatable`, I'd suggest also `IExternalDelegatableSymbol` as the interface name